src/Model/Term.php

```diff
@@ -4,20 +4,145 @@
  */
 namespace Friendica\Model;
 
+use Friendica\Core\System;
+use Friendica\Database\DBM;
 use dba;
 
-require_once "include/dba.php";
+require_once 'boot.php';
+require_once 'include/dba.php';
 
 class Term
 {
+	public static function insertFromItemId($itemid)
```

Comment:
In accordance with my comment below, it would be possibly better to call this function ```insertTagFromItemId```.

Reply:
There's no concept of `tags` in this class, only `terms`, according to the database table name. I'd rather not introduce any confusion about what's what.

Reply:
Maybe `insertFromTagFieldByItemId`?

Reply:
Done.